dags/aggregates_python_helpers.py

```diff
@@ -280,26 +280,35 @@ def assign_terc_and_validate(row, gdf1):
 
     terc_code = row['terc']
     untypical = None  # a new variable for new statuses like 'Unknown' or 'Unknown2'
+    jednostki_numer = row['jednostki_numer']
```

Comment:
is there any dvantage over using terc than jednostki_numer?

Reply:
Sometimes, when "terc" column is empty for a selected record, it can happen that in the same time it is possible to extract the code value from the value located in "jednostki_numer" column.